### DISCUSSION TITLE
What is the basic memory and cpu requirment

### DISCUSSION THREAD

**Discussion by sauravazad** [strapi/strapi#6227] [Legacy Topics]
Hey  
I have  a couple of question :-
  - Have anyone performed any bench-marking regarding RPS
  - What is the minimum memory requirement
  - Has anyone run it on any of the serverless platform

**Comment by robertpiosik** [comment]
> Have anyone performed any bench-marking regarding RPS

What do you mean exactly? Requests per seconds? Well, your app will be as fast as you write it to be;) What strapi does by its convenieces you would likely do very similarly yourself - it's just a thin layer on top of Koa Node.js framework. Expensive queries will always be slow, but with proper caching, for example on HTTP layer with Varnish, your app will be very resilient.

> Has anyone run it on any of the serverless platform

You can launch Strapi on any `Node.js runtime` (just search for this phrase, I won't promote any option here).

> What is the minimum memory requirement

It really depends on your use case - how complex queries you make, whether you cache static responses and so one. Running PM2 process of your application will take a few dozen megabytes, rest depends on what you do with incoming requests.

**Reply by derrickmehaffy** [reply]
memory requires would be 2GB of ram to build, beyond that depends on the size of the project. Mine which has ~94 models uses about half a gig (give or take) at a time with some fairly deep nesting

**Reply by sauravazad** [reply]
> memory requires would be 2GB of ram to build, beyond that depends on the size of the project. Mine which has ~94 models uses about half a gig (give or take) at a time with some fairly deep nesting

Thanks for the insights. I am trying to get this up and running using GCP Cloud functions.
